lib/engine.py

```diff
@@ -654,15 +664,21 @@ def insert_statement(self, values):
         columncount = len(self.table.get_insert_columns(join=False, create=False))
         insert_stmt = "INSERT INTO " + self.table_name()
         insert_stmt += " (" + columns + ")"
-        insert_stmt += "VALUES ("
-        for i in range(0, columncount):
-            insert_stmt += "%s, "
-        insert_stmt = insert_stmt.rstrip(", ") + ");"
-        n = 0
-        while len(values) < insert_stmt.count("%s"):
-            values.append(self.format_insert_value(None, types[n]))
-            n += 1
-        insert_stmt %= tuple([str(value) for value in values])
+        insert_stmt += " VALUES " 
+        vals = ""
```

Comment:
Unclear what this line is doing. Is `valk` below supposed to be `vals`? If so I don't see the need to set it to `""` here since it will get overwritten by `valk = rows`.